### DISCUSSION TITLE
p/better-infinite-scroll

### DISCUSSION THREAD

**Discussion by giscus** [OU9999/ou9999-dev#18] [General]
# p/better-infinite-scroll

오늘의집과 같은 무한스크롤 구현 과정과 후기

https://ou9999-dev.com/p/better-infinite-scroll

**Comment by taekyeomlee** [comment]
안녕하세요! 글 정말 잘 읽었습니다.

저도 최근에 react-virtuoso를 사용하여 프로젝트에서 리스트 가상화 작업을 진행했습니다. 저 역시 스크롤 복원과 관련된 고민이 있었는데, 공식 문서를 살펴보다가 라이브러리 차원에서 해결할 수 있는 방법이 있다는 것을 알게 되었습니다.

제가 작성한 이 정보가 도움이 될 것 같아 공유드립니다. [링크](https://github.com/petyosi/react-virtuoso/discussions/1066)

**Reply by OU9999** [reply]
좋은 글 공유 감사드립니다!

작성해주신 정보 정말 좋은것같습니다
